### ISSUE TITLE
XSS issue in filename

### ISSUE THREAD

**Issue by timovn** [broncowdd/BoZoN#129]
Try uploading a file with name like `abs">def.jpg`  or worse : `abc" onmouseover="alert('boom')">.jpg`

Code is interpreted in the page.
Filename should be escaped (at least) or anything else than alpha-num should be striped when server gets the file.

**Comment by broncowdd** [comment]
Curieux, je n'ai qu'une erreur de GD... mais pas d'interprétation du 
code ?!

**Comment by timovn** [comment]
Effectivement, avec un fichier vierge renommé en `abc">def` il affiche une erreur `[application/octet-stream, 0] The file doesn't fit `.

Par contre, prends une image (jpg ou autre) et donne lui comme nom  `abc">def` (avec ou sans extension).

Ça marche aussi avec d’autres fichiers non-vide (je viens d’y arriver avec un PDF).
![screenshot at 2016-03-31 22 14 00](https://cloud.githubusercontent.com/assets/4772201/14189505/f37b013c-f78d-11e5-83ea-b09f6f11dc7d.png)
